### DISCUSSION TITLE
Verificação de arquivos já inseridos na mesma pasta

### DISCUSSION THREAD

**Discussion by kauetsu** [iterate-ch/cyberduck#14712] [Q&A]
Ao realizar a transferência de uma pasta com diversos arquivos dentro e outras pasta, o cyberduck esta começando novamente a transferência e substituindo ou comparando os arquivos, ele não pula os arquivos que já estão nas pastas.

**Comment by kauetsu** [comment]
Pasta 1 com 5 pastas dentro e 5 arquivos em cada pasta. Ao realizar a transferência do drive do google para o drop box ele não esta analisando se a pasta já foi criada e se os arquivos já estão nelas, ele começa a copiar tudo novamente todas as vezes.

**Comment by panda2134** [comment]
Google translation (hope this helps):

Verification of files already inserted in the same folder

When transferring a folder with several files inside and other folders, cyberduck is starting the transfer again and replacing or comparing the files, it does not skip the files that are already in the folders.

Folder 1 with 5 folders inside and 5 files in each folder. When performing the transfer from the google drive to the drop box it is not analyzing if the folder has already been created and if the files are already in them, it starts to copy everything again every time.

**Comment by dkocher** [comment]
Choose _Skip_ in the [overwrite prompt](https://docs.cyberduck.io/cyberduck/transfer/#overwrite-prompt) when transferring files.
